packages/auth/openapi/id-provider.yaml

```diff
@@ -99,7 +99,7 @@ paths:
                 type: object
                 properties:
                   access:
-                    $ref: ./schemas.yaml#/access
+                    $ref: ./schemas.yaml#/components/schemas/access
```

Comment:
Heads up, this is overlapping with:
- https://github.com/interledger/rafiki/pull/692